app1/src/components/cards/UsersListCard.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Card from '@mui/material/Card';
+
+export default function MainCard({userTask}) {
```

Comment:
I like that you created a component to render all task but You never have to use name 2 files with different name but actually they are the same component. I mean you should use : on components/MainCard ---> export default function MainCard() {}.  on componentes/UsersListCard ---> export default function UserListCard() {}.   Right now you are using the same MainCard for both folders, you never have to do that because others developer won't know the meaning.